test/examples/Addition.ju

```diff
@@ -0,0 +1,22 @@
+let nat = %Micheslon.nat
```

Comment:
```suggestion
let nat = %Michelson.nat
```

Reply:
Huh. Nice catch Chris. @mariari any idea why the typo parsed?

Reply:
OK I checked the test results. It parses the primitives name as they are without checking if they're valid. 

Later, we can start thinking about matching them at this parse stage and spit out error messages that suggest the user may want something that has similar spelling.